packages/sdk/src/conversation/notification.browser.ts

```diff
@@ -382,3 +383,75 @@ export class TeamsBotInstallation implements NotificationTarget {
     );
   }
 }
+
+/**
+ * Provide static utilities for bot notification.
+ *
+ * @remarks
+ * Only work on server side.
+ *
+ * @example
+ * Here's an example on how to send notification via Teams Bot.
+ * ```typescript
+ * // initialize (it's recommended to be called before handling any bot message)
+ * const notificationBot = new NotificationBot(adapter);
+ *
+ * // get all bot installations and send message
+ * for (const target of await notificationBot.installations()) {
+ *   await target.sendMessage("Hello Notification");
+ * }
+ *
+ * // alternative - send message to all members
+ * for (const target of await notificationBot.installations()) {
+ *   for (const member of await target.members()) {
+ *     await member.sendMessage("Hello Notification");
+ *   }
+ * }
+ * ```
+ *
+ * @beta
+ */
+export class NotificationBot {
+  private static readonly conversationReferenceStoreKey = "teamfx-notification-targets";
+  private static conversationReferenceStore: ConversationReferenceStore;
+  private static adapter: BotFrameworkAdapter;
+
+  /**
+   * Initialize bot notification.
+   *
+   * @remarks
+   * Only work on server side.
+   *
+   * To ensure accuracy, it's recommended to initialize before handling any message.
+   *
+   * @param adapter - the bound `BotFrameworkAdapter`
+   * @param options - initialize options
+   *
+   * @beta
+   */
+  public static initialize(adapter: BotFrameworkAdapter, options?: NotificationOptions) {
```

Comment:
Keep the definition the same as non-browser ones.

Reply:
Good catch! Fixed already.